digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadNameService.scala

```diff
@@ -120,7 +120,7 @@ class RoadNameService() {
         if (currentRoadNames.isEmpty) {
           val projectRoadNames = ProjectLinkNameDAO.get(roadNumber, projectID)
           if (projectRoadNames.isEmpty) {
-            return None
+            Some(Map("roadName" -> None, "isCurrent" -> false))
```

Comment:
Since we don't need to return at any time a `None` we should remove from the `getRoadNameByNumber` method the Option.

Reply:
Option[] and Somes() removed from the method.